lib/accounts.js

```diff
@@ -444,24 +444,10 @@ export class Accounts {
    * @param {Account} account - Account to update
    * @returns {Promise<Account>}
    * @tag Accounts
+   * @deprecated PUT call to accounts is deprecated, please use PATCH.
```

Comment:
Let's change the wording on this to "update call for accounts is deprecated, please use the patch call."